_posts/2016-12-30-Is-not-a-Sacrament-is-just-a-stand-up.markdown

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title:  "Is not a Sacrament, is just a stand-up"
+date:   2016-12-30 11:28:00 -0300
+categories: software, stand-up, scrum, en
+---
+
+Each day, at 12.00 am, the Minister is ready to start the Sacrament. It doesn’t really matter what you did or not the last day or even the quality of the code, the only most important thing is that you be there on the stand-up and talk about something.
+
+Bosses, PMs, whatever has to re-organize his priority, there are at least 36 different most important things in a project than this meeting. 
```

Comment:
Algunas correcciones:
..the only most important thing is that you **are** there..
Bosses, PMs, **whateever**, **have** to re-organize..
..there are at least 36 different **more** important things..

Muy bueno